src/components/PatchesPage/PatchCard.tsx

```diff
@@ -45,16 +48,19 @@ export const PatchCard: React.FC<Props> = ({
   id,
   description,
   createTime,
+  author,
+  authorDisplayName,
   projectID,
   projectIdentifier,
   status,
+  type,
   builds,
   canEnqueueToCommitQueue,
   isPatchOnCommitQueue,
   analyticsObject,
 }) => {
   const createDate = new Date(createTime);
-
+  const verb = type == "project" ? "by" : "on";
```

Comment:
it's okay style to add this ternary to jsx instead of assigning to a variable

Reply:
yeah it more readable having this inline.